frontend/src/app/organization/organization-offers-list/organization-offers-list.component.scss

```diff
@@ -0,0 +1,22 @@
+@import '~bootstrap/scss/bootstrap-grid';
+$mediumgray-light:            #CCD1D9 !default;
+
+.crop-circle {
```

Comment:
As mentiond in the html template comment, I would write something like this:

```
.volontulo-link-circle {
  display: block;
  width: 34px;
  height: 34px;
  border-radius: 50%;
  background-size: cover;
  background-position: 50% 50%;
}

td:first-child {
  vertical-align: middle;
}
```
Lot less code with the same effect. Also, as a bonus I am leveraging the fact that we are inside of a table cell, so td:first-child rule centers the image links in the first column.

Reply:
In fact, it is better now, because in previous version images were cut from top and bottom, so they weren't circle, just like it is in backend version.